src/zui/ZUIEditTextInPlace/index.tsx

```diff
@@ -25,6 +24,14 @@ const useStyles = makeStyles((theme) => ({
     fontSize: 'inherit !important',
     fontWeight: 'inherit',
   },
+  showBorder: {
+    border: '2px dotted transparent',
+    borderColor: lighten(theme.palette.primary.main, 0.65),
+    borderRadius: 10,
+    paddingLeft: 10,
+    paddingRight: 0,
+    transition: 'all 0.2s ease',
+  },
```

Comment:
There should be no need for this extra class, as there is already styling for the border in the `input` class.